src/components/Tables/TableFilterAndExtraColumnsBar.tsx

```diff
@@ -64,12 +63,36 @@ export const filtersToComponents = <Data,>(
     });
 };
 
-const TableFilterAndExtraColumnsBar = <Data,>(props: Props<Data>): React.ReactElement => {
+const TableFilterAndExtraColumnsBar = <Data, Filter extends CoreFilter>(
+    props: Props<Data, Filter>
+): React.ReactElement => {
+    const handleClear = (): void => {
```

Comment:
I moved the definition of this function inside this component, instead of passing it as a prop from Table